src/patternfly/layouts/Page/page.scss

```diff
@@ -230,25 +186,21 @@
 
 // Sidebar
 .pf-l-page__sidebar {
-  grid-area: 2 / 1;
   z-index: var(--pf-l-page__sidebar--ZIndex);
-  width: var(--pf-l-page__sidebar--Width);
+  flex: 1;
+  order: initial;
+  width: var(--pf-l-page__sidebar--md--Width); // hardcode sidebar based on design spec, must match pf-n-nav--Width
+  max-width: var(--pf-l-page__sidebar--md--Width); // max-width value set for browser to calculate max-width while transitioning. sharing value to keep consistent
+  grid-area: 2 / 1;
   padding-top: var(--pf-l-page__sidebar--PaddingTop);
   padding-bottom: var(--pf-l-page__sidebar--PaddingBottom);
   overflow-x: hidden;
   overflow-y: auto;
   background-color: var(--pf-l-page__sidebar--BackgroundColor);
+  box-shadow: var(--pf-l-page__sidebar--BoxShadow);
```

Comment:
This is also applied to `.pf-m-expanded` - does it need to be both places?

Reply:
we could change this, but it's the way it was built. There is three states to the nav. One with `pf-m-expanded`, one with `pf-m-collapsed` and one without any class as the default which needs the shadow. I talked to @TheRealJon about this today actually. It could be worth rethinking a bit.